stdlib/number.gr

```diff
@@ -417,14 +443,19 @@ let chebyshevSine = (radians: Number) => {
  * @returns The computed sine
  *
  * @since v0.5.2
+ * @history v0.5.3: Handle NaN and Infinity
  */
 export let sin = (radians: Number) => {
-  let quot = reduceToPiBound(radians)
-  let bounded = radians - pi * quot
-  if (quot % 2 == 0) {
-    chebyshevSine(bounded)
+  if (isNaN(radians) || isInfinite(radians)) {
```

Comment:
Why did you decide to start using these functions here but not in the functions above?